src/basic/macro.h

```diff
@@ -488,4 +488,14 @@ static inline size_t size_add(size_t x, size_t y) {
         return y >= SIZE_MAX - x ? SIZE_MAX : x + y;
 }
 
+
+#define TRACE_POINT(name, dev, ...)                                             \
+        do {                                                                    \
+                const char *__s  = NULL;                                        \
+                DeviceAction __a = _DEVICE_ACTION_INVALID;                      \
```

Comment:
c compiler says double underscores are owned by compiler. maybe just use a single one here, should be enough too